src/codetypes.h

```diff
@@ -55,22 +54,22 @@ typedef int32_t qcoord_t;
 // #define RAD2DEG            (180.0/PI)
 // #define DEG2RAD            (PI/180.0)
 #define TOL 1.0e-10
-#define HOLEMAPSIZE 192
+enum : uint8_t { HOLEMAPSIZE = 192 };
 // #define NFRINGE            3
 // #define NVAR               6
-#define WALLNODETYPE 0
-#define OUTERNODETYPE 1
+// #define WALLNODETYPE 0
+// #define OUTERNODETYPE 1
+enum : uint8_t { WALLNODETYPE = 0, OUTERNODETYPE = 1 };
 /*==================================================================*/
 /* ADAPTIVE HOLE MAP OCTANT INFO                                    */
 /*==================================================================*/
-#define INTERSECT_ALG                                                          \
-    1 // [0] point-box inclusion only
-      // [1] face-box intersection (water-tight)
-
-#define NON_UNIQUE_NODES                                                       \
-    1 // [0] wbc nodes are NOT listed as obc nodes
-      // [1] wbc nodes may also be listed as obc nodes
+//    1 // [0] point-box inclusion only
+//      // [1] face-box intersection (water-tight)
+enum : uint8_t { INTERSECT_ALG = 1 };
 
+//    1 // [0] wbc nodes are NOT listed as obc nodes
+//      // [1] wbc nodes may also be listed as obc nodes
+enum : uint8_t { NON_UNIQUE_NODES };
```

Comment:
I think this needs a value of 1.

Reply:
It appears that I overlooked this. It's fixed now.